src/README.md

```diff
@@ -85,13 +85,13 @@ the schema will be filtered out in the final output.
 
 ### Overview
 The following diagram summarizes the architecture:
-![](data/architecture.png)
+![](./data/architecture.png)
 
 ## Publish
 Data tables are made available for use via the `publish.py` script which uploads the files to a file
-server and creates the different versions of the datasets (like date subsets, the master table, JSON
+server and creates the different versions of the datasets (like date subsets, the main table, JSON
 formatted files, etc.). Data is published automatically by the CI server with every change to the
-master branch; to run the publish step locally, simply execute this script from the `src` directory:
+main branch; to run the publish step locally, simply execute this script from the `src` directory:
```

Comment:
Is the branch name actually changing with this PR?  I don't see a "main" branch -- will this be created?

Reply:
Yes, we will create a `main` branch after this is merged. The reason why I'm merging to `master` first is so we can see the diff and comment on the PR.